tests/js/utils.js

```diff
@@ -111,7 +113,9 @@ export function WithTestRegistry( { children, callback, registry = createTestReg
 
 	return (
 		<RegistryProvider value={ registry }>
-			{ children }
+			<FeaturesProvider value={ enabledFeatures }>
+				{ children }
+			</FeaturesProvider>
```

Comment:
If we need to include the provider here to prevent hooks from breaking, the value should be empty (array/object) rather than using the normal import so that features always need to be enabled explicitly like they already are in the provider passed in the `children`.

Alternatively, `enabledFeatures` could come from a prop to `WithTestRegistry` to simplify the need for additional wrappers in the consumer, but I'm not sure if that's a good idea 🤔  What do you think?

Reply:
I think long-term we probably want to rename the `WithTestRegistry` to something more like `WithRootOptions` or `WithTestOptions` or something. We can then have a `features` prop and a `registry` prop, etc. That makes more sense.

But for now adding the `features` option to `WithTestRegistry` is nice, let's do that 👍🏻